src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof module !== 'undefined') {
+  module.exports = window.WebComponents;
+}
```

Comment:
Why was this hack needed again? There are no examples here so it shouldn't be needed.